src/App.tsx

```diff
@@ -1,24 +1,299 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 11361;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [updatedTodos, setUptatedTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [sortBy, setSortBy] = useState<string>('all');
+  const [isTodoDeleted, setIsTodoDeleted] = useState(false);
+  const [todosNotCompleted, setTodosNotCompleted] = useState(0);
```

Comment:
There is no need for this to be a state, this could be a value memoized with useMemo based on `todos`